sentimental/pom.xml

```diff
@@ -44,6 +44,13 @@
       <artifactId>twitter4j-core</artifactId>
       <version>[4.0,)</version>
     </dependency>
```

Comment:
Hey! I still can't run your stuff on my computer but I don't want to keep holding you back because I'm certain this works fine. The only thing is that Kevin pushed the datastore stuff and GitHub says you have a merge conflict because you don't have the datastore dependency here, so just add that and I'll approve your PR

Reply:
Thanks for checking, I will add that dependency as soon as I can and I will let you know.